local.properties

```diff
@@ -0,0 +1,11 @@
+## This file is automatically generated by Android Studio.
+# Do not modify this file -- YOUR CHANGES WILL BE ERASED!
+#
+# This file must *NOT* be checked into Version Control Systems,
+# as it contains information specific to your local configuration.
+#
+# Location of the SDK. This is only used by Gradle.
+# For customization when using a Version Control System, please read the
+# header note.
+#Fri Mar 24 21:07:17 GMT 2017
+sdk.dir=/Users/bhalchandrawadekar/Library/Android/sdk
```

Comment:
I suspect this file should not be committed. It contains settings that are specific to your machine. Do you want to add a line to .gitignore for this file and remove it from your PR?